examples/falling-balls/settings.gradle.kts

```diff
@@ -1 +1,8 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
```

Comment:
I intentionally removed that earlier to have a test that works without custom settings. Why it was needed?